opendistro-for-elasticsearch/opendistro-build
# [HELM] Kubernetes 1.22: Deprecated apiVersion

*Issue #, if available:*

Kubernetes Ingress apiVersion: extensions/v1beta1 is deprecated and will be removed in 1.22
Kubernetes Ingress apiVersion: rbac.authorization.k8s.io/v1beta1 is deprecated and will be removed in 1.22

*Description of changes:*

Use the new APIVersion. (Fix: https://github.com/opendistro-for-elasticsearch/opendistro-build/issues/779)

*Test Results:*

- Deploy using the following chart values on Kubernetes 1.20 :
```
kibana:
  ingress:
    enabled: true
    hosts:
      - "kibana.test.fr"
    pathType: Prefix

elasticsearch:
  client:
    ingress:
      enabled: true
      hosts:
        - "client.test.fr"
      pathType: Prefix
```
- Check the Ingress and Roles (With https://github.com/FairwindsOps/pluto)

**Note: If this PR is related to Helm, please also update the README for related documentation changes. Thanks.**
**https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/README.md**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on signing off your commits, please check [here](https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/main/CONTRIBUTING.md#sign-your-work).
